governance-proposal/TSC-Charter-Draft.md

```diff
@@ -0,0 +1,103 @@
+
+Changes to TSC membership should be posted in the agenda, and may be suggested as any other agenda item.
+
+No more than one-fourth of the TSC members may be affiliated with the same employer. If removal or resignation of a TSC member, or a change of employment by a TSC member, creates a situation where more than one-fourth of the TSC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more TSC members affiliated with the over-represented employer(s).
+
+The TSC members shall consist of Maintainers from Core Projects as defined in the project lifecycle document and Section 7.
+
+The TSC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On Air, or through any other appropriate means selected by the TSC). The meeting shall be directed by the TSC Chairperson. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
+
+## Section 5. Responsibilities of the TSC.  
+
+Subject to such policies as may be set by the Board, the TSC is responsible for all technical development within the Node.js Foundation, including:
+
+* Setting release dates
+* Release quality standards
+* Technical direction
+* Project governance and process (including this policy)
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+* Development process and any coding standards
+* Mediating technical conflicts between Collaborators or Foundation projects
+
+The TSC will define Node.js Foundation’s release vehicles and serve as Node.js Foundation’s primary technical liaison body with external open source projects, consortiums and groups.
+
+## Section 6. Node.js Foundation Operations.
+
+The TSC will establish and maintain a development process for Node.js Foundation Projects. The development process will establish guidelines for how the developers and community will operate. It will, for example, establish appropriate timelines for TSC review (e.g. agenda items must be published at least a certain number of hours in advance of a TSC meeting).
+
+There will be multiple Projects under the Node.js Foundation organized by modules or subsystems. The TSC is responsible for organizing the Project structure, including possibly the creation and alignment of sub-Projects. Each Project must be within such policies as may be set by the Board, have a well-defined scope and must work within that scope. The development process will provide for Projects to follow the lifecycle process as described in the Project Lifecycle document. The development process will include a process for the TSC to oversee and approve changes in the lifecycle of a Project, which will include consideration of the following criteria:
+
+* Cleanliness of code base
```

Comment:
@Fishrock123 this document is specific to the TSC which is the final arbiter of what goes in to core, not in to other projects which have some level of autonomy. The project lifecycle doc may note something about code cleanliness but I would expect that each project have its own policy around that rather than trying to impose the policy for core on all other projects.